app.rb

```diff
@@ -69,7 +69,7 @@
           issue.set_release(nil)
         end
 
-        issue.add_pull_request(pull_request.raw_data['html_url'])
+	issue.add_pull_request(pull_request.raw_data['html_url']) if !pull_request.cherry_pick?
```

Comment:
Omg, why is that first indentation 3 spaces!? lol

This is fine, but more Rubyist to use `unless`.